tool/internal_pkg/tpl/client.go

```diff
@@ -80,6 +83,10 @@ type {{.ServiceName}}_{{.RawName}}Client interface {
 func NewClient(destService string, opts ...client.Option) (Client, error) {
 	var options []client.Option
 	options = append(options, client.WithDestService(destService))
+
+	{{if and .IsExternal (not .HasStreaming)}}
```

Comment:
IsExternal can be removed because the code generation logic is only for external kitex.

Reply:
@jayantxie I don't want to generate this code for internal kitex. Do you think it's ok to rename IsExternal -> IsInternal and generate code when not .IsInternal?

Reply:
Actually this template won't be executed for internal kitex, so there's no need to add this option.

Reply:
@jayantxie It's executed for internal kitex as well, internal kitex uses this template and also adds some extension to this template.